snap/snapcraft.yaml

```diff
@@ -70,6 +79,7 @@ parts:
       - libqt5core5a
       - libqt5network5
       - libbluray2
+      - qt5-gtk-platformtheme
```

Comment:
This library is not present in the kde-frameworks content snap, so I think you're right to include it here!